### DISCUSSION TITLE
icons for mail and packages

### DISCUSSION THREAD

**Discussion by eon0613** [moralmunky/Home-Assistant-Mail-And-Packages#889] [Q&A]
how to i change the box icons to shipper icons like UPS, FEDEX..etc

![image](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/assets/129685917/33adf99b-541a-4b3c-8cbf-a17836535bc8)

**Comment by firstof9** [comment]
That'd be a question for [the custom card repo](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages-Custom-Card).

**Comment by eon0613** [comment]
ok thank you
